Source/WebKit/UIProcess/WebFramePolicyListenerProxy.cpp

```diff
@@ -62,7 +62,6 @@ void WebFramePolicyListenerProxy::didReceiveAppBoundDomainResult(std::optional<N
 void WebFramePolicyListenerProxy::didReceiveSafeBrowsingResults(RefPtr<BrowsingWarning>&& safeBrowsingWarning)
```

Comment:
This should probably add an early return if we've already received safe browsing results for this listener.  Otherwise the second call might confuse the state.